spec

```diff
@@ -0,0 +1,12532 @@
+
+beginseg
+    name "kanji"
+    include "build/baserom/kanji.o"
+endseg
+
+beginseg
+    name "link_animetion"
+    romalign 0x1000
+    include "build/baserom/link_animetion.o"
+endseg
+
+beginseg
+    name "icon_item_static_old"
+    romalign 0x1000
+    include "build/baserom/icon_item_static_old.o"
+endseg
+
+beginseg
+    name "icon_item_24_static_old"
+    romalign 0x1000
+    include "build/baserom/icon_item_24_static_old.o"
+endseg
+
+beginseg
+    name "icon_item_field_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/icon_item_field_static.o"
+endseg
+
+beginseg
+    name "icon_item_dungeon_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/icon_item_dungeon_static.o"
+endseg
+
+beginseg
+    name "icon_item_gameover_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/icon_item_gameover_static.o"
+endseg
+
+beginseg
+    name "icon_item_jpn_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/icon_item_jpn_static.o"
+endseg
+
+beginseg
+    name "icon_item_vtx_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/icon_item_vtx_static.o"
+endseg
+
+beginseg
+    name "map_i_static"
+    include "build/baserom/map_i_static.o"
+endseg
+
+beginseg
+    name "map_grand_static"
+    include "build/baserom/map_grand_static.o"
+endseg
+
+beginseg
+    name "item_name_static"
+    include "build/baserom/item_name_static.o"
+endseg
+
+beginseg
+    name "map_name_static"
+    include "build/baserom/map_name_static.o"
+endseg
+
+beginseg
+    name "icon_item_static_test"
+    include "build/baserom/icon_item_static_test.o"
+endseg
+
+beginseg
+    name "icon_item_24_static_test"
+    include "build/baserom/icon_item_24_static_test.o"
+endseg
+
+beginseg
+    name "schedule_dma_static_old"
+    include "build/baserom/schedule_dma_static_old.o"
+endseg
+
+beginseg
+    name "schedule_dma_static_test"
+    increment 0x1000
+    include "build/baserom/schedule_dma_static_test.o"
+endseg
+
+beginseg
+    name "schedule_static"
+    compress
+    include "build/baserom/schedule_static.o"
+endseg
+
+beginseg
+    name "story_static"
+    compress
+    romalign 0x1000
+    include "build/baserom/story_static.o"
+endseg
+
+beginseg
+    name "do_action_static"
+    romalign 0x1000
+    include "build/baserom/do_action_static.o"
+endseg
+
+beginseg
+    name "message_static"
+    romalign 0x1000
+    include "build/baserom/message_static.o"
+endseg
+
+beginseg
+    name "message_texture_static"
+    romalign 0x1000
+    include "build/baserom/message_texture_static.o"
+endseg
+
+beginseg
+    name "nes_font_static"
+    romalign 0x1000
+    include "build/baserom/nes_font_static.o"
+endseg
+
+beginseg
+    name "message_data_static"
+    romalign 0x1000
+    include "build/baserom/message_data_static.o"
+endseg
+
+beginseg
+    name "staff_message_data_static"
+    romalign 0x1000
+    include "build/baserom/staff_message_data_static.o"
+endseg
+
+beginseg
```

Comment:
Similarly here, should the code seg have more spacing to match how boot is done, or should boot have less?